modules/4337/certora/harnesses/Account.sol

```diff
@@ -53,6 +53,59 @@ contract Account is Safe {
     function getValidUntilTimestamp(bytes calldata sigs) external pure returns (uint48) {
         return uint48(bytes6(sigs[6:12]));
     }
+
+    /*
+    This function is used to calculate the hash of the signatures in a standard way. This will result in signatures having possible excess data to result in the same hash
+    as the same signatures without the excess data. This is required for formal verification.
+
+    Actual Signature:
+    "0x" +
+    "0000000000000000000000000000000000000000000000000000000000000001 0000000000000000000000000000000000000000000000000000000000000041 00" + // encoded EIP-1271 signature 1
+    "0000000000000000000000000000000000000000000000000000000000000004 deadbeef"                                                              // length of bytes + data of bytes of signature 1
+
+    With Excess Data:
+    "0x" +
+    "0000000000000000000000000000000000000000000000000000000000000001 0000000000000000000000000000000000000000000000000000000000000041 00" + // encoded EIP-1271 signature 1
+    "0000000000000000000000000000000000000000000000000000000000000004 deadbeef"                                                              // length of bytes + data of bytes of signature 1
+    "ffffff"                                                                                                                                 // excess data
+
+    Both will have the same canonical hash: 0xc860b1a81652620308a8138a17ef5105d9b18e6e766516ffd3de9897260d1320
```

Comment:
We can also link to the Safe documentation pages (since this won't be on-chain anyway).

Reply:
Added 👍🏾